lib/features/children/overview/pages/family_overview_page.dart

```diff
@@ -89,6 +90,12 @@ class FamilyOverviewPage extends StatelessWidget {
           body: SafeArea(
             child: buildFamilyOverviewBody(state, context),
           ),
+          floatingActionButtonLocation:
+              FloatingActionButtonLocation.centerFloat,
+          floatingActionButton: (state is FamilyOverviewUpdatedState &&
+                  (state.hasChildren || !state.isAdultSingle))
+              ? const DownloadGivtForKidsAppWidget()
+              : null,
```

Comment:
The banner is shown even if the user set up children and parents. You can see it in design. It should show if the user clicked "I will do this later" on the success screen or if the user dismissed it manually. Probably needs some local flag i guess 🤔 
![Overview](https://github.com/givtnl/givt-app/assets/43034761/dac4de77-fe10-4e11-bcd4-0ff0665e1d48)